search-resources-api/src/main/java/no/unit/nva/search/RequestUtil.java

```diff
@@ -45,13 +51,32 @@ public static SortOrder getSortOrder(RequestInfo requestInfo) {
         return SortOrder.fromString(requestInfo.getQueryParameters().getOrDefault(SORTORDER_KEY, DEFAULT_SORT_ORDER));
     }
 
+    public static UriWrapper getRequestUri(RequestInfo requestInfo) {
+        String path = getRequestPath(requestInfo);
+        String domainName = getRequestDomainName(requestInfo);
+        return new UriWrapper(HTTPS, domainName).addChild(path);
```

Comment:
do not return the UriWrapper. return the URI. The UriWrapper is actually a class containing the supplied URI. Creating a class is almost for free and we do not pay much for creating many UriWrappers when we need them. Better to exchange URIs and if we need to manipulate them we can use UriWrappers now or something else later.